clients/src/main/java/org/apache/kafka/server/quota/ClientQuotaCallback.java

```diff
@@ -1,4 +1,5 @@
 /*
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout comment separator -->
---
RSPEC-2176: Class has same name as interface
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._